templates/geosafe/analysis/dynamic_scripts/create/main.html

```diff
@@ -369,7 +369,13 @@
         $.get('{% url 'geosafe:layer-tiles' %}', {'layer_id': layer_id}, function (data) {
 
             //default use of the marker
-            var marker = L.marker([data['layer_bbox_y0'], data['layer_bbox_x0']], {
+            center_y = data['layer_bbox_y0'] + ((data['layer_bbox_y1'] - data['layer_bbox_y0'] ) / 2);
+            center_x = data['layer_bbox_x0'] + ((data['layer_bbox_x1'] - data['layer_bbox_x0'] ) / 2);
+            corner_1 = L.latLng(data['layer_bbox_y0'], data['layer_bbox_x0'])
+            corner_2 = L.latLng(data['layer_bbox_y1'], data['layer_bbox_x1'])
+            var marker = L.marker(L.latLngBounds(corner_1, corner_2).getCenter(), {
```

Comment:
Hmm, if you have this, why bother calculating center at the top?

Reply:
ooppsss... i forgot to delete the variables of `center_x` and `center_y`.
should i push another PR to fix this @lucernae ?